content/change/teaching/resources.md

```diff
@@ -1,13 +1,16 @@
-title: Teaching resources
+title: Resources
 
 # Where can I find resources for teaching open science?
 
 Open Science is here to stay. Therefore, students deserve to be prepared for a future where reproducable research practices are standard. We have found that students intuitively understand the necessity for changes to the scientific practice, and enjoy applying state-of-the-art methods in their projects.
 
 As with scientific resources, members of the Open Science community have created and share freely their teaching materials. Here are a few resources:
 
-* The [**Open Science Training Handbook**](https://book.fosteropenscience.eu) is a fantastic and comprehensive resource that covers many topics around open science. It's built for people who would like to create open science training, and very in-depth.
 * Many researchers have made their [**course syllabi for open and reproducible methods**](https://osf.io/vkhbt/) openly available.
+* At the SIPS Meeting 2018, a Hackathon collected a whole bunch of [teaching resources](https://osf.io/bts5d/wiki/home/), including syllabi, videos, podcasts, journal articles and much more.
+* The [Framework for Open and Reproducible Research Training](https://forrt.netlify.com) presents recommendations to improve the adoption of open science teaching practices by modifying the academic incentive structure.
 * The [Society for the Improvement of Psychological Science](http://improvingpsych.org/) has collected [**teaching and training materials**](https://osf.io/ehpt4/) in general, as well as a collection of [**modular course materials for research methods**](https://osf.io/zbwr4/wiki/home/). For example, ...
     * The [Collaborative Replications and Education Project](https://osf.io/wfc6u/wiki/home/) provides teaching materials around a set of five studies that undergraduate students can replicate, with the aim of publishing the resulting data when enough has been gathered.
     * The [Project Teaching Integrity in Empirical Research](https://www.projecttier.org/tier-classroom) (TIER) develops methods and tools for enhancing research transparency through teaching. These can be used by faculty who teach quantitative methods or supervise student research or by students who want to adopt transparent and replicable research practices independetly.
+
+* If you teach a lab course, you might be interested in these [slides provided by the LMU Munich](https://osf.io/mgwk8/).
```

Comment:
Would it make sense to link to the first set of slides (https://osf.io/b8qr2/), and note that they are in German?